src/localization/langCodes.ts

```diff
@@ -797,6 +849,10 @@ const languageCodes: { [Key: string]: ILanguage} = {
     name: 'Portuguese',
     nativeName: 'Português'
   },
+  'pt-pt': {
```

Comment:
Ja lokalizēts ir pliks valodas kods bez reģiona, nevajadzētu ņemt to kā fallback šajā gadījumā? Piem., ja parādītos jauna MT sistēma pt-BR, un sarakstā nav 'pt-BR', tad skatās, vai gadījumā nav 'pt'

Reply:
nu principā varētu tā

Reply:
pievienoju tādu variantu.

Reply:
izskatās ka tagad vienmēr meklēs tikai pēc sākumdaļas, tad pazūd jēga no lokalizētiem valodu kodiem ar reģionu. Doma bija to atmest reģionu tikai tad, ja nav pieejams nosaukums pilnajam kodam

Reply:
taisnība, izlaboju